applet/src/keyb.c

```diff
@@ -44,7 +44,7 @@ void reset_backlight()
     // enabling backlight again.
     void (*f)(uint32_t,uint32_t) = (void*)( 0x802b80a + 1 ); // S: ??? 0x0802BAE6
     f(0x40020800,0x40);
-#else if defined(FW_S13_020)
+#elseif defined(FW_S13_020)
```

Comment:
src/keyb.c: In function 'reset_backlight':
src/keyb.c:47:2: error: invalid preprocessing directive #elseif
 #elseif defined(FW_S13_020)
  ^